packages/server/src/server.ts

```diff
@@ -661,11 +661,18 @@ app.useRpc('eth_maxPriorityFeePerGas', async () => {
  */
 
 app.useRpc('debug_traceTransaction', async (params: any) => {
-  const transactionHash = params[0];
-  const tracer = params[1].tracer;
-  const tracerConfig = params[1].tracerConfig;
-  return logAndHandleResponse('debug_traceTransaction', [transactionHash, tracer, tracerConfig], (requestId) =>
-    relay.eth().debugService().debug_traceTransaction(transactionHash, tracer, tracerConfig, requestId),
+  const transactionIdOrHash = params[0];
+  const { tracer, ...otherParams } = params[1];
+
+  let tracerConfig: object;
+  if (tracer === TracerType.CallTracer) {
+    tracerConfig = otherParams?.tracerConfig ?? { onlyTopCall: false };
+  } else {
+    tracerConfig = otherParams ?? { disableMemory: false, disableStack: false, disableStorage: false };
```

Comment:
I am not sure if we want to enable `storage` by default. This might make the requests really slow if no tracer options are passed. What do you think 🤔?

Reply:
Also, it's good to note that the options which remix passes are slightly different from what is described in our HIP: https://github.com/ethereum/remix-project/blob/master/libs/remix-debug/src/trace/traceManager.ts#L63

Reply:
Hmm this is a cherry pick PR against release/0.50 so changes are not ideal. Please open PR against main for any desired changes.

Reply:
Yeah, it's still totally fine to release it as it is now.

It's just something that came up to me today and is probably good to discuss it at some point and decide if we want that behaviour.